WcaOnRails/app/controllers/api/v0/competitions_controller.rb

```diff
@@ -8,4 +7,13 @@ def show
     end
     render json: competition
   end
+
+  def results
+  	competition = Competition.where(id: params[:id], showAtAll: true).firsts
+  	unless Competition
+  		render json: { error: "Competition with id #{params[:id]} not found" }, status: 404
+  		return
+  	end
+  	render json: Result.search_by_competition(competition.id, params: params)
+  end
```

Comment:
> scope :visible, -> { where(showAtAll: true) }

I've been thinking that `showAtAll` **really** needs to be renamed to `visible`, since the name is so misleading. This is a really nice way to do it without touching the schema!